.github/workflows/tests.yml

```
@@ -0,0 +1,100 @@
+        run: npm ci
+
+      - name: Run tests
+        id: jest
+        run: npm test -- --coverage --runInBand
+
+      - name: Append coverage summary
+        if: success() && steps.jest.outcome == 'success'
+        run: |
+          if [ -f coverage/coverage-summary.json ]; then
+            node <<'NODE'
+const fs = require('fs');
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 54: could not find expected ':'

(syntax-check)